google-services/Location/BasicLocationSample/BasicLocationSample/packages.config

```diff
@@ -3,16 +3,19 @@
   <package id="Xamarin.Android.Arch.Core.Common" version="1.0.0" targetFramework="monoandroid80" />
   <package id="Xamarin.Android.Arch.Lifecycle.Common" version="1.0.1" targetFramework="monoandroid80" />
   <package id="Xamarin.Android.Arch.Lifecycle.Runtime" version="1.0.0" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Animated.Vector.Drawable" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Annotations" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Compat" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Core.UI" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Core.Utils" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Fragment" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Media.Compat" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.v4" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.v7.AppCompat" version="26.1.0.1" targetFramework="monoandroid80" />
-  <package id="Xamarin.Android.Support.Vector.Drawable" version="26.1.0.1" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Animated.Vector.Drawable" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Annotations" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Compat" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Core.UI" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Core.Utils" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Design" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Fragment" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Media.Compat" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Transition" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.v4" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.v7.AppCompat" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.v7.RecyclerView" version="26.1.0" targetFramework="monoandroid80" />
+  <package id="Xamarin.Android.Support.Vector.Drawable" version="26.1.0" targetFramework="monoandroid80" />

```

Comment:
We should continue to use version `26.1.0.1` for all Xamarin.Android.Support packages, rather than `26.1.0`.

Reply:
Changed.